incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -603,38 +651,45 @@ Explorer](https://cloud.google.com/monitoring/charts/metrics-explorer) by
 searching the metrics keywords (e.g. `namespace_conditions`).
 
 In order to monitor metrics via Stackdriver:
+1. Save your some key information as environment variables. You may adjust these
+   values to suit your needs; there's nothing magical about them.
+   ```bash
+   GSA_NAME=hnc-metric-writer
+   PROJECT_ID=my-gcp-project
+
+   ```
 1. Enable Workload Identity (WI) on either a
    [new](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster)
    or
    [existing](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_an_existing_cluster)
    cluster.
-2. Install HNC as described [above](#admin-install).
-3. [Create a Google service account (GSA)](https://cloud.google.com/docs/authentication/production#creating_a_service_account):
+1. Install HNC as described [above](#admin-install).
```

Comment:
why all 1?

Reply:
It's better Markdown style, especially if you have to insert new elements, which is what I did. So I could have either left all the numbers be wrong, corrected them all, or set them all to "1" which is a common Markdown idiom.

Reply:
oh i see thanks for the explanation.